### ISSUE TITLE
History of compressive footwear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9915]
**Preferred term label:**
History of compressive footwear

**Synonyms**
Tight shoes, toe-box shoes, high heel shoes

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Morton neuroma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000302
- label: History of compressive footwear
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:20934103
- def: A past medical history of wearing tight shoes, toe-box shoes, or high heel shoes.
- comment: This may be a risk factor for Morton neuroma.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9915
- synonyms:
